### PR TITLE
Wait for raf after all mouse events.

### PR DESCRIPTION
**Pull request by JoelEinbinder**
#122

### CODE REVIEW

File: lib/Mouse.js [hunk 1]
```diff
@@ -41,6 +41,8 @@ class Mouse {
       x, y,
       modifiers: this._modifiersMask()
     });
+    // This is a hack for now, to make clicking less race-prone. See crbug.com/747647
+    await this.evaluate(() => new Promise(f => requestAnimationFrame(f)));
```
Comment (aslushnikov) [inline lib/Mouse.js]: is this needed? AFAIU the mouseMove does wait for ack: [input_handler.cc](https://cs.chromium.org/chromium/src/content/browser/devtools/protocol/input_handler.cc?q=input_handler.cc&sq=package:chromium&dr&l=403)

Comment (JoelEinbinder) [reply lib/Mouse.js]: In practice it behaves otherwise, at least as far as I can tell.

**Comment by aslushnikov** [comment]
Exploring the codesearch, I failed to find the place where the mousemove swallows the ACK. It might be that we're facing something else here - let's investigate before landing

File: lib/Mouse.js [hunk 2]
```diff
@@ -69,7 +73,11 @@ class Mouse {
       clickCount: (options.clickCount || 1)
     });
     // This is a hack for now, to make clicking less race-prone. See crbug.com/747647
-    await this.evaluate(() => new Promise(f => requestAnimationFrame(f)));
+    await this._client.send('Runtime.evaluate', {
+      expression: 'new Promise(f => requestAnimationFrame(f))',
```
Comment (aslushnikov) [inline lib/Mouse.js]: let's do rafraf!

**Pull request merged**